manifests/kustomize/third-party/postgresql/base/kustomization.yaml

```diff
@@ -2,7 +2,8 @@ apiVersion: kustomize.config.k8s.io/v1beta1
 kind: Kustomization
 resources:
 - pg-deployment.yaml
+- pg-pv.yaml
```

Comment:
I shouldn't need a PV resource, it is automatically created by PVC when it is bound to a deployment resource.

Reply:
I'm not sure about this. I added the PV file because the PVC had trouble creating due to lack of PV. Maybe I was doing something wrong during my deployment. Could you give me more reference on this? Thank you.

Reply:
You can look at the `Events` for the PVC to learn more about why is PV not created. Maybe the disk driver has something wrong when provisioning a PV. But we shouldn't create PV resource manually in codebase. Refer  to MySQL manifest folder: https://github.com/kubeflow/pipelines/tree/master/manifests/kustomize/third-party/mysql/base

Reply:
You are right about this. I tried a fresh deployment without the PV, and it passed the tests. The previous error was likely caused by something else. I removed the pv declaration.